edx_when/models.py

```diff
@@ -98,8 +98,10 @@ def actual_date(self):
         """
         if self.abs_date:
             return self.abs_date
-        else:
+        elif self.rel_date:
             return self.content_date.policy.actual_date + self.rel_date
+        else:
+            return None
```

Comment:
Why not have this return `self.content_date.policy.actual_date`? Then you don't need the conditional in api.py

Reply:
Yeah, that's a good call.